### PR TITLE
Add kubevirt-ssp-operator deployment

### PR DESCRIPTION
**Pull request by yanirq**
**What this PR does / why we need it**:
Use Ansible to orchestrate kubevirt-ssp-operator deployment while OLM continues to develop.

**Release note**:

```release-note
Launch kubevirt-ssp-operator 
```

### CODE REVIEW

File: roles/kubevirt-ssp/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#TODO change default namespace?
+kubevirt_ssp_namespace: "{{ kubevirt_ssp_image_namespace | default('kube-system') }}"
```
Comment (MarSik) [inline roles/kubevirt-ssp/defaults/main.yaml]: Shouldn't the default be kubevirt now?

File: roles/kubevirt-ssp/tasks/deprovision.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+- name: Check that kubevirt-ssp-operator.yml still exists in /tmp
+  stat:
```
Comment (MarSik) [inline roles/kubevirt-ssp/tasks/deprovision.yml]: Why don't you use the https://docs.ansible.com/ansible/2.4/tempfile_module.html module?

Comment (MarSik) [reply roles/kubevirt-ssp/tasks/deprovision.yml]: Is that because you want to avoid instantiating the template again when executed again?

Comment (yanirq) [reply roles/kubevirt-ssp/tasks/deprovision.yml]: I was using the same convention as in other playbooks in the project, but I can change it accordingly

Comment (MarSik) [reply roles/kubevirt-ssp/tasks/deprovision.yml]: Oh, lets keep it then, but I can't say I like this style..

File: roles/kubevirt-ssp/defaults/main.yaml (hunk 1 above)
Comment (rthallisey) [reply roles/kubevirt-ssp/defaults/main.yaml]: Should be kubevirt by default

File: roles/kubevirt-ssp/tasks/provision.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- name: Create Kubevirt-SSP operator
+  command: "{{ cluster_command }} apply -f /tmp/kubevirt-ssp-operator.yml"
+
+- name: Create Kubevirt-SSP
+  block:
+  - name:
+  command: "{{ cluster_command }} apply -f /tmp/kubevirt_v1_kubevirtssp_crd.yaml"
+#  - name:
+#  command: "{{ cluster_command }} apply -f /tmp/kubevirt_v1_kubevirtssp_cr.yaml"
```
Comment (rthallisey) [inline roles/kubevirt-ssp/tasks/provision.yml]: remove comment

**Pull request closed without merging**